tracr/io/dicom/read.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+"""
+A script that converts dicom files to an intensity voxel array
+The resulting array is 1000 x 1024 in cross section direction and 1014 "tall"
+"""
+
+import sys, os
+import numpy as np
+import dicom
+
+def read(dcm_files):
+	# Import image and details. Input is a list of filename strings
+	layer = dicom.read_file(dcm_files[0])
+	dicom_array = np.zeros((int(layer.Rows), int(layer.Columns), len(dcm_files)))
+	pixel_size = float(layer.PixelSpacing[0])*1000
```

Comment:
Indexing PixelSpacing is curious. Does it store both x and y pixel dimensions, but you tacitly assume they will always be the same?